### ISSUE TITLE
Login with Home Assistant is not working

### ISSUE THREAD

**Issue by Klagio** [estevez-dev/ha_client#442]
I can't access to Hass anymore, now I just updated the app, but same problem.

Setting has worked fine for many months until today

**Comment by estevez-dev** [comment]
Did it stop working before or after update?

**Comment by rchovan** [comment]
Hi, I have same problem, I was using HomeAssistant in python environment, on static public IP with LetsEncrypt certificate and DNS. I was able to connect to my HA with HA Client. But When I migrated my installation to Hass.io, I can't connect anymore from HA Client. Everything is same, static public IP, username, password, DNS name, even LE certificate is same. I can connect from browser without problem, even on Android. 

When I fill login credentials and press NEXT, screen changes to Initialization, and then again back to login, where I see credentials fields again.

This are logs from HA client:
```
1105 12:34:08 [Debug] :  AppLifecycleState.inactive
1105 12:34:08 [Debug] :  AppLifecycleState.resumed
1105 12:34:08 [Debug] :  AppLifecycleState.inactive
1105 12:34:08 [Error] :  Loading settings...
1105 12:34:08 [Error] :  Long-lived token read successful
1105 12:34:08 [Debug] :  Launching OAuth
1105 12:34:09 [Debug] :  AppLifecycleState.paused
1105 12:35:07 [Debug] :  AppLifecycleState.inactive
1105 12:35:07 [Debug] :  AppLifecycleState.resumed
```

When I fill correct credentials, I can't see any error in Hass.io logs, but when I fill wrong credentials, I see error in notification panel:
```
Login attempt failed
Login attempt or request with invalid authentication from x.x.x.x
```

**Comment by julupanter** [comment]
Same problem as rchovam.
I'm on a fresh install on a Raspberry Pi 4 with Hassio on docker.
My only 'extras' is DuckDns addon for SSL.
 I was trying your app but then I face this login loop.

**Comment by estevez-dev** [comment]
Very strange behavior. Will try to figure out why it happening.
Meanwhile you can (generate long-lived token)[http://ha-client.homemade.systems/docs#manual-long-lived-token] and use it in Connection settings of HA Client.

**Comment by julupanter** [comment]
> 
> 
> Very strange behavior. Will try to figure out why it happening.
> Meanwhile you can [generate long-lived token](http://ha-client.homemade.systems/docs#manual-long-lived-token) and use it in Connection settings of HA Client.

The long-lived token worked for me.

Thank you!

**Comment by estevez-dev** [comment]
Hey @rchovan , is it still an issue for you? If yes, please tell me which device do you use and what Android version is on it. Thanks.

**Comment by rchovan** [comment]
Hi, I was using token to sign in. It working correctly. When I have tried login with username and password it still does not work. 
An another bug, when i logoff i see this in settings page. 

![Screenshot_20191127-211703](https://user-images.githubusercontent.com/6713920/69756917-e29f2c80-115b-11ea-8d30-ef19b8c305d4.jpg)


My device is Samsung Galaxy S8 with Android 9.

**Comment by Stele77** [comment]
I can confirm the same problem. Xiaomi Mi 9T Official EU ROM, Android 10. 
The Login page is in a loop.
I can login fine with all other Apps like Hasskit and Home Assistant App.

**Comment by estevez-dev** [comment]
Authentication with webview will be back in 0.8.0. It should resolve most of the issues with login.

**Comment by estevez-dev** [comment]
Closing as fixed after 0.8.0 with webview. Feel free to reopen if needed
